docs/AQL/master03-syntax.adoc

```diff
@@ -239,6 +239,31 @@ FROM
 WHERE
    o/data[at0002]/events[at0003]/data/items[at0015]/items[at0018]/items[at0019]/items[at0021]/name/defining_code/code_string matches {'18919-1', '18961-3', '19000-9'}
 --------
++
+Value list data is enclosed within curly braces after a matches operator. Each item is separated by comma if there are multiple items in the list. Value list items can be string, date time, integer, or real. Quotes are required for strings and date times.
++
+A value list is only used in AQL `WHERE` clause when the criteria is to match one item of the list. The relationships among these value list items are "OR".
++
+Some examples of value list:
++
+--------
+matches {'string item 1', 'string item 2', 'string item3'}  // a string list. It Equivalent to
+                                                            // matches  string item1' or ..
+                                                            // matches 'string item 2' or ..
+                                                            // matches 'string item 3'
+matches {'20060101', '20070101', '20080101'}                // a date value list
```

Comment:
We should convert this to '2006-01-01' etc; we do allow both forms in openEHR but the extended (with dashes for dates and ':' for times) is strongly preferred. So better to have examples that follow our own rules...!

Reply:
@wolandscat change was done and added a small comment about the recommendation